app/controllers/accounts/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Accounts::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
+  # def new
+  #   super
+  # end
+
+  # POST /resource/confirmation
+  # def create
+  #   super
+  # end
+
+  # GET /resource/confirmation?confirmation_token=abcdef
+  # def show
+  #   super
+  # end
+
+  # protected
+
+  # The path used after resending confirmation instructions.
+  # def after_resending_confirmation_instructions_path_for(resource_name)
+  #   super(resource_name)
+  # end
+
+  # The path used after confirmation.
+  # def after_confirmation_path_for(resource_name, resource)
+  #   super(resource_name, resource)
+  # end
```

Comment:
What is this file doing?

Reply:
All the files under controller/accounts/ are generated by devise , these files are provided in order to add any extra functionality to devise SignIn/SignUp Flow. These files stays commented unless we want to add any user desired functionality in these controllers as they are inherited from the Devise Controllers.
These controllers were generated using 
rails generate devise:controllers accounts